### PR TITLE
Enable travis-ci

### PR DESCRIPTION
**Pull request by Nugine**
This pr adds 6 checks:

|arch|os|rust|
|-|-|-|
|AMD64|ubuntu 16.04|msrv|
|Arm64|ubuntu 16.04|msrv|
|AMD64|ubuntu 16.04|stable|
|Arm64|ubuntu 16.04|stable|
|AMD64|ubuntu 16.04|nightly|
|Arm64|ubuntu 16.04|nightly|

Build status:

<https://travis-ci.com/github/datenlord/datenlord>

References:

1. <https://docs.travis-ci.com/user/languages/rust/>
2. <https://docs.travis-ci.com/user/reference/linux/>
3. <https://docs.travis-ci.com/user/reference/osx/>

### CODE REVIEW

**Comment by Nugine** [comment]
Travis CI failed because of mismatched types. (compile error)
See <https://travis-ci.com/github/datenlord/datenlord/jobs/396944281>

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
     branches: [master]
 
 env:
-  CI_RUST_TOOLCHAIN: 1.46.0
+  CI_RUST_TOOLCHAIN: stable
```
Comment (pwang7) [inline .github/workflows/ci.yml:7]: let's fix it to 1.46 for now

**Review by pwang7 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Nugine) [reply .github/workflows/ci.yml]: Ok

**Comment by Nugine** [comment]
Travis CI failed on arm64.
GitHub CI failed on macos.
Waiting for the master branch to fix.

**Comment by Nugine** [comment]
ready to merge

**Comment by Nugine** [comment]
Clippy still failed on macOS.
Waiting for the master branch to fix.

**Review by pwang7 — APPROVED** [review]

**Pull request merged**